src/app/static/src/tests/components/forms/dynamicForm.test.ts

```diff
@@ -32,12 +34,30 @@ describe('Dynamic form component', function () {
         ]
     };
 
+    const formMetaWithAllControlTypes = {
+        ...fakeFormMeta
+    };
+
+    formMetaWithAllControlTypes.controlSections[1].controlGroups[0].controls.push({
+            name: "id_3",
+            type: "multiselect",
+            options: [{id: "opt1", label: "option 1"}],
+            required: false
+        } as SelectControl,
+        {
+            name: "id_4",
+            type: "select",
+            options: [{id: "opt2", label: "option 2"}],
+            required: false
+        } as SelectControl);
+
     it("renders form with id", () => {
         const rendered = mount(DynamicFormComponent, {
             propsData: {
                 formMeta: fakeFormMeta,
                 id: "test-id"
-            }
+            },
+            sync: false
```

Comment:
This `sync: false` everywhere is due to a bug in vue test-utils. It seems to be triggered when doing a full mount where a child component has a watcher/computed property, so only surfaced in this test suite once I added the multiselect. Bizarely you have to add "sync: false" to every mount or shallow mount in the suite to stop it happening, even if it only affects one test in the suite. https://github.com/vuejs/vue-test-utils/issues/1130

Reply:
Weird! Candidate for pulling that shallowMount out into a common method, maybe with partialPropsData as a parameter (and include `formMeta: fakeFormMeta` in the props in the method, as I think that's in all the instances?